apis/install/v1alpha1/executor_types.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/armadaproject/armada-operator/apis/common"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
This file was changed so I could get the code to build correctly with lint and tests.  

The executor code is not functional as is but I'd like to at least get the CI sooner rather than later.